library/src/main/java/com/google/android/exoplayer/text/eia608/Eia608Parser.java

```diff
@@ -153,6 +156,12 @@
         continue;
       }
 
+      // Ignore repeated control characters.
+      if (ccData1 == previousCtrl1 && ccData2 == previousCtrl2) {
```

Comment:
Isn't this check too broad? Per the spec, you should only ignore repeated control characters if ccData1 is in the range [0x10, 0x1F]. You should also not ignore a second repeat (e.g. if you see the same control characters three times in a row).